src/jit/jitstd/vector.h

```diff
@@ -439,10 +439,7 @@ void vector<T, Allocator>::clear()
     {
         m_pArray[i].~T();
     }
-    m_allocator.deallocate(m_pArray, m_nCapacity);
-    m_pArray = NULL;
```

Comment:
`vector::clear()` is not required to release the memory. Doing this prevents memory reuse.